tests/network/test_node_storage.py

```diff
@@ -32,6 +38,16 @@ def test_one_node_stores_a_bunch_of_others(federated_ursulas, ursula_federated_t
     assert not newcomer.known_nodes
 
     newcomer.start_learning_loop(now=True)
+    def start_lonely_learning_loop():
+        newcomer.start_learning_loop()
```

Comment:
This pattern has started to bug me (I know I'm the one who initially introduced it).  I'll think of something better.  It's good for now.

Reply:
Actually - Im not entirely sure how this was re-introduced? Wasn't this deprecated in #574?

Reply:
Ahh, yes you're right.  I'm not sure?

In any case, we still need a higher-level test for "not now" starting of the learning loop.  Or at least, I think we do? 

But yeah, the immediate question is on the matter of how this has been reintroduced.  (And actually, does it reliably pass?)

Reply:
As far as I can tell, Yes - It passes every time.